src/index.spec.js

```diff
@@ -131,7 +131,7 @@ describe('Mongo', () => {
 
   describe('findUserById', () => {
     it('should return null for not found user', async () => {
-      const ret = await mongo.findUserById('unknowuser');
+      const ret = await mongo.findUserById('589871d1c9393d445745a57c');
```

Comment:
But the usage of `findUserById` should be valid, because your test want to check a non-existing user, rather test the validation. 
The `toMongoID` should  throw an exception in case of invalid object id.